core/src/main/java/io/confluent/connect/storage/schema/StorageSchemaCompatibility.java

```diff
@@ -69,25 +98,54 @@ protected boolean check(
   FULL;
 
   private static final Map<String, StorageSchemaCompatibility> REVERSE = new HashMap<>();
+  private static final Set<SimpleImmutableEntry<Schema.Type, Schema.Type>> PROMOTABLES;
```

Comment:
Really wish we could use this from `SchemaProjector` rather than duplicating the code